cooldowns_druid.lua

```diff
@@ -251,6 +251,12 @@ LCT_SpellData[99] = {
 	specID = { SPEC_DRUID_GUARDIAN },
 	cooldown = 30
 }
+LCT_SpellData[99] = {

```

Comment:
Please add the name in a comment above it

Reply:
Accurate name is actually Incapacitating Roar, updated.

Reply:
Make it talent = true for all specs? It is actually baseline for guardian but it's not that common in arena anyways.

Reply:
But for feral/balance/resto, it's a very important CC ability that are often used to chain into a cyclone

Reply:
> Make it talent = true for all specs? It is actually baseline for guardian but it's not that common in arena anyways.

I guess so, yes.
Other way would be `talent = { specId, ... }`, but there's no support for that yet.

Is there a way we can detect the talent beforehand? If we see them using another bear affinity spell (or bear spec spell, I guess the former is a subset of the latter) we can detect the talent.

Reply:
I don't think there is a way to detect talent until something is used for the first time.
Making it talent = true for bear spec wouldn't be that bad, bear spec is very niche in arena, and it will show up correctly after first usage, which will most likely be within the first 10 sec of the game.

Reply:
> I don't think there is a way to detect talent until something is used for the first time.

No, I rather mean: Bear Affinity also gives access to Thrash & Frenzied Regen.
We can detect they have Incapacitating Roar if they use either of those (similarly to how charges to bop/freedom work).

Reply:
specID = { SPEC_DRUID_GUARDIAN, SPEC_DRUID_BALANCE, SPEC_DRUID_FERAL, SPEC_DRUID_RESTO },
talent = { spell ID of thrash/regen}?
Then we don't know whether it's baseline from bear spec or from affinity talent.

I just wrote it as talent = true for all specs for now.